Description: Skip partmap test if qemu-img doesn't exist (e.g. on the Hurd)
Author: Colin Watson <cjwatson@debian.org>
Forwarded: no
Last-Update: 2010-11-22

Index: b/tests/partmap_test.in
===================================================================
--- a/tests/partmap_test.in
+++ b/tests/partmap_test.in
@@ -51,6 +51,8 @@
     echo
 }
 
+which qemu-img >/dev/null 2>&1 || exit 77
+
 imgfile=`mktemp "${TMPDIR:-/tmp}/tmp.XXXXXXXXXX"` || exit 1
 outfile=`mktemp "${TMPDIR:-/tmp}/tmp.XXXXXXXXXX"` || exit 1
 
